packages/http/src/forwarder/HttpForwarder.ts

```diff
@@ -63,20 +65,17 @@ export class HttpForwarder implements IForwarder<IHttpOperation, IHttpRequest, I
     }, toError);
   }
 
-  private updateHostHeaders(baseUrl: string, headers?: IHttpNameValue) {
-    // no headers? do nothing
-    if (!headers) return headers;
-
-    // host header provided? override with actual hostname
-    if (headers.hasOwnProperty('host')) {
-      return {
-        ...headers,
-        host: new URL(baseUrl).host,
-        forwarded: `host=${headers.host}`,
-      };
-    }
+  private static updateHostHeaders(baseUrl: string, headers: IHttpNameValue = {}) {
```

Comment:
I'm ok with not having this as an instance method — but I think instead of making it static let's just put it out of the way of the class and leave it in the file somewhere else so nobody uses it outside of this file (which is good)